app/main.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(self, name: str,
+                 health: int = 100,
+                 hidden: bool = False
+                 ) -> None:

```

Comment:
btw, it's a bad code style.
Good:
```python
("a", "b", "c", "d")
```
Also good:
```python
(
"a",
"b",
"c",
"d"
)
```

Bad:
```python
(
"a", "b"
"c", "d"
)
```
Also bad:
```python
("a",
"b",
"c",
"d"
)
```

Reply:
thanks, thats look batter)